src/repositories/Versionable/VersionableRepository.ts

```diff
@@ -52,11 +52,20 @@ export default class  VersionableRepository <D extends mongoose.Document , m ext
          console.log(id , ' user is deleted');
          return await this.modelType.findByIdAndUpdate(id , {deletedAt: new Date()});
     }
-  public async findall() {
-      return await this.modelType.find();
+  public async findall(skip, limit, sort) {
+     // tslint:disable-next-line: radix
+     const a =  parseInt(skip);
+      // tslint:disable-next-line: radix
+      const b = parseInt(limit);
+     if (sort === undefined) {
+         return await this.modelType.find().skip(a).limit(b).sort({updatedAt: -1});
+     }
+      return await  this.modelType.find().skip(a).limit(b).sort(sort);
   }
+
```

Comment:
don't check for 'undefined' just check sort is there or not if not then use sort({updatedAt: -1}) else .sort(sort)